core/busybox/files/acpid.run

```diff
@@ -1,2 +1,3 @@
 #!/bin/sh
-exec acpid -fl
+[ -d /etc/acpi ] || mkdir -p /etc/acpi
```

Comment:
This can just be `mkdir -p /etc/acpi`. Also, it's probably simpler to create this directory in the package itself.

Reply:
While it can be created in the package itself, I figured it's optional since a computer can run fine without `acpid` running. Besides, files and directories inside `/etc` aren't deleted.